### DISCUSSION TITLE
Add aria-label to Figma components

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#1437] [Proposals]
## Description
Include to components de tag Aria-label to make possible defines aria label from design teams

Current components with aria-label in mistica-web:
- Avatar
- Box
- Boxed
- Callout
- Cards (all)
- Checkbox
- Empty state card
- Empty state
- Icon button
- Maybe-dismissable
- Breadcrumbs
- Progress bar
- Stepper
- Theme
- Touchable
- Advanced Data Card
- Blocks (all)

**Comment by yceballost** [comment]
POC: https://www.figma.com/file/WTjzIwqdXmGBCsUbzHEIG1/POC---Add-aria-label-to-Figma-components?type=design&node-id=0-2639&mode=design

**Comment by aweell** [comment]
Vamos a hacer un research con equipos desarrollo y diseño para evaluar si es una feature util a la hora de implementar un producto digital.
